scripts/Muon/GUI/ElementalAnalysis2/context/context.py

```diff
@@ -4,13 +4,40 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
+from typing import List
+from Muon.GUI.Common.muon_load_data import MuonLoadData
 
 
-class ElementalAnalysisContext(object):
+class DataContext(object):
```

Comment:
Its ok for now, but in the future you may want to split these into their own files. Once the length of the classes starts growing it will become difficult to find things if it is all in one file.

Reply:
I've split them out as part of #29382 as the context file is getting longer.